src/deck-options.md

```diff
@@ -474,6 +480,18 @@ If you're first switching from SM-2 and still wish to use this option, we recomm
 
 Compute minimum recommended retention (CMRR) attempts to find the desired retention value that leads to the most material learnt, in the least amount of time. The calculated number can serve as a reference when deciding what to set your desired retention to. You may wish to choose a higher desired retention, if you’re willing to trade more study time for a greater retention rate. However, setting your desired retention lower than the minimum is not recommended, as you'll spend more time studying than necessary, due to increased forgetting.
 
+### The Simulator
+
+In the latest version of Anki, you can use the simulator to get an estimate of your workload,
+either in reviews per day or in minutes of studying per day.
+**Days to simulate** controls the duration of the simulated study history. **Additional new cards to simulate** 
+controls whether the simulator should simulate more cards than what already exists under this preset. For example, if you currently have 100 cards under this preset,
+and you set **Additional new cards to simulate** to 50, the simulator will simulate a total of 150 cards. **New cards/day** and **Maximum reviews/day** control
+how many new cards will be learned each day and the maximum number of reviews per day. **Maximum interval** controls the maximum interval length (in days).
+To make the simulation as realistic as possible, the simulator takes into account real memory states (difficulty, stability, retrievability) of your cards as opposed
```

Comment:
```suggestion
To make the simulation as realistic as possible, the simulator takes into account the real memory state (difficulty, stability, retrievability) of your cards.
```

Reply:
Each card has its own memory state, so the plural form (states) makes more sense, no?

Reply:
I'm not a grammar expert, but to me it reads more naturally, as the word is attached to the singular 'memory', and sentences should read the same if parenthetical passages are removed.

Reply:
And having written that, I'm now second-guessing myself. If nobody agrees, feel free to leave it. :-)

Reply:
It's about count noun versus mass noun. So, you wouldn't say states of affair because state is mass noun here. But, in our case I feel memory state is a countable thing and not an uncountable singular concept.